CSS.css

```diff
@@ -58,6 +59,16 @@ ul.nav li.active a {
 a {
   color: white;
 }
+=======
+body.homepage {
+    background-image: url('images/Background.jpg');
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-size: cover;
+    background-color: #464646;
+    color: #FCFBE3;
+  }
+>>>>>>> parent of 8f0475d... Removed to-do list from website code
```

Comment:
These lines:

* `<<<<<<< HEAD`
* `=======`
* `+>>>>>>> parent of 8f0475d... Removed to-do list from website code`

Shows there is a conflict & should not be committed back unresolved